include/seqan/simd/simd_base_seqan_impl_avx2.h

```diff
@@ -146,9 +146,6 @@ _fillVector(TSimdVector & vector,
                     std::get<0>(args)
                 )
             );
-#else
```

Comment:
I dont understand these changes. Was the problem related to intel or gcc49 and if this was only changed to accomodate for gcc49 we can revert it because we dropped that for simd, didnt't we?

Reply:
It is related to gcc4.9. They use a macro for expanding to the intrinsic function. Hence the pack expansion does not work as it is happening after the macro expansion and that fails, as the preprocessor sees only one argument.
But, yes let revert it, if gcc 4.9 is dropped for SIMD anyway.